### ISSUE TITLE
error when building the project 

### ISSUE THREAD

**Issue by lord-maul** [MaxBittker/sandspiel#69]
I found out that there are several dependencies needed to be installed while the documentation hasn't listed them, including Rust, wasm-pack and cargo-watch.

the first three build commands were OK, but after I opened a new terminal and typed "cargo watch -s 'wasm-pack build'", there occurred an error saying: 
"Found argument 'build'' which wasn't expected, or isn't valid in this context" 

Can anyone help me with this? I've done a lot of searching but no results. Many thanks.

**Comment by MaxBittker** [comment]
did you run that command from the crate directory?

**Comment by MaxBittker** [comment]
sorry the project is hard to build

**Comment by lord-maul** [comment]
> did you run that command from the crate directory?

I've tried but came out with the same error. I go through the docs at wasm-pack but I can't get any clue.

I've also tried to run just  `wasm-pack build` from the crate directory. It succeeded. And the `wasm-pack pack` command also succeeded, but the `wasm-pack publish` failed as it seemed cannot be pulished to npm.

Here's a pic of how it shows after I have run all that command. The canvas turns out to be black.
![result](https://user-images.githubusercontent.com/27283476/58422988-9e58a400-80c6-11e9-99cd-ee7b034fdbf2.png)
